preupg/application.py

```diff
@@ -581,31 +580,23 @@ def _set_devel_mode(self):
         else:
             self._devel_mode = 0
 
-    def _check_available_contents(self):
-        cnt = 0
-        is_dir = lambda x: os.path.isdir(os.path.join(self.conf.source_dir, x))
-        dirs = os.listdir(self.conf.source_dir)
-        self.list_scans = []
-        for dir_name in filter(is_dir, dirs):
-            if SystemIdentification.get_assessment_version(dir_name):
-                self.conf.scan = dir_name
```

Comment:
Here, the function was changing the self.conf.scan, which is basically available from anywhere of the code Preupgrade Assistant. I don't think function with _check_ in its name should be doing such a nastiness. So instead of changing it, I'm returning it.

Also this for loop does basically the same as the already existing _list_contents()_ function. So I used it.